### ISSUE TITLE
'Others Icons size' sometimes get changed unexpectedly...

### ISSUE THREAD

**Issue by whalley** [moneymanagerex/moneymanagerex#4241]
MMEX version:
 - [x] 1.5.x
 - [ ] 1.6.x

https://forum.moneymanagerex.org/viewtopic.php?p=22787#p22787

A few people are reporting that the 'Others icon size' setting seems to change without them doing anything to cause it. I've not experienced this but there may be a lurking bug somewhere, which is only triggered occasionally.

I think the cause of the iconsize changing is maybe here... 
```
void mmGUIFrame::OnBeNotified(wxCommandEvent& /*event*/)
{
    Model_Setting::instance().Set(INIDB_NEWS_LAST_READ_DATE, wxDate::Today().FormatISODate());
    wxLaunchDefaultBrowser(mmex::weblink::News);

    int toolbar_icon_size = Option::instance().getToolbarIconSize();
    Option::instance().setIconSize(toolbar_icon_size);
    toolBar_->SetToolBitmapSize(wxSize(toolbar_icon_size, toolbar_icon_size));
    toolBar_->SetToolNormalBitmap(MENU_ANNOUNCEMENTMAILING, mmBitmap(png::NEWS));

    const auto b = toolBar_->FindById(MENU_ANNOUNCEMENTMAILING);
    if (b) b->SetShortHelp(_("Register/View Release &Notifications"));
}
```

It does not reset the IconSize back after the call to set the size to the toolbar_icon_size
`Option::instance().setIconSize(toolbar_icon_size);`

Actually I think this code and the similar code in mmframe.cpp can be amended to specify the size explicitly when calling mmBitMap.

**Comment by tactilis** [comment]
Apart from the size of the status icons in the transaction lists, what else does the 'Others icon size' control now?

If I recall, some time ago we discussed the variable size of icons on dialogs and you updated MMEX so that these are now fixed size.

So, what's left for 'Others icon size'?

'Others' is a bit of a cryptic name. Should we rename it?

Should there be, as suggested in [https://forum.moneymanagerex.org/viewtopic.php?p=22785](https://forum.moneymanagerex.org/viewtopic.php?p=22785) a setting that controls the row height (and font size) of the transaction list and automatically scales the icon size appropriately?
